[set User as foreign key]
You need import the User model at the top of your file
 from django.contrib.auth.models import User
so python can resolve the reference.
